resources/definitions/fdmprinter.def.json

```diff
@@ -511,6 +511,16 @@
                     "settable_per_extruder": false,
                     "settable_per_meshgroup": false
                 },
+                "machine_start_gcode_first":
+                {
+                    "label": "Start GCode must be first",
+                    "description": "Do not add any gcodes before the start gcode block.",
```

Comment:
Try to also include when and why one should use this. 

So i'd personally word it a bit more like

"This setting controls if the start-gcode is forced to always be the first g-code that is done. Without this it can happen that other g-code, such as a T0 can be inserted before the start g-code"

Reply:
Thank you for the example I believe I have this fixed now